website/docs/docs/build/metricflow-commands.md

```diff
@@ -54,31 +39,34 @@ You can install [MetricFlow](https://github.com/dbt-labs/metricflow#getting-star
 
 **Note**, you'll need to manage versioning between dbt Core, your adapter, and MetricFlow.
 
-</TabItem>
+Something to note, MetricFlow `mf` commands return an error if you have a Metafont latex package installed. To run `mf` commands, uninstall the package.
 
+</TabItem>
 </Tabs>
 
-Something to note, MetricFlow `mf` commands return an error if you have a Metafont latex package installed. To run `mf` commands, uninstall the package.
-
 ## MetricFlow commands
 
 MetricFlow provides the following commands to retrieve metadata and query metrics. 
 
 <Tabs>
-<TabItem value="cloud" label="Commands for dbt Cloud CLI">
+<TabItem value="cloudcommands" label="Commands for dbt Cloud">
 
-You can use the `dbt sl` prefix before the command name to execute them in the dbt Cloud CLI. For example, to list all metrics, run `dbt sl list metrics`. For a complete list of the MetricFlow commands and flags, run the `dbt sl --help` command in your terminal.
+You can use the `dbt sl` prefix before the command name to execute them in the dbt Cloud IDE or dbt Cloud CLI. For example, to list all metrics, run `dbt sl list metrics`. For a complete list of the MetricFlow commands and flags, run the `dbt sl --help` command in your terminal. 
```

Comment:
So the sad reality is that at the moment `dbt --help` does not work in IDE due to how they process logs. If you do `dbt build --help` or `dbt sl list metrics --help` or any other help command, it will just output nothing...

Reply:
Might be worth talking to someone from the IDE team if we can get support for that.

Reply:
For now tho, the user's best bet is to refer to the docs.